src/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -1230,8 +1243,7 @@
                :where [:= :certname certname]
                :order-by [[:producer_timestamp :desc]]
                :limit 1}]
-    (sql/with-query-results result-set (hcore/format query)
-      (:producer_timestamp (first result-set)))))
+    (:producer_timestamp (first (jdbc/query (hcore/format query))))))
```

Comment:
we're using query here and in a couple other spots while it seems like we're using query-with-resultset in most other similar places. I don't mind, but is there any particular reason?